cmd/start-ssh-agent.cmd

```diff
@@ -73,6 +73,8 @@ if exist "%GIT%" (
         set "HOME=%USERPROFILE%"
         "!SSH_ADD!"
         echo.
+    ) else (
+        set "HOME=%USERPROFILE%"

```

Comment:
This has no effect as HOME is not being used further down in the script, and we are in a setlocal / endlocal block here anyway.